src/routes/politicians/[id]/votelog/+page.svelte

```diff
@@ -151,7 +151,7 @@
 			<!-- TODO: Add link -->
 			<a class="text-text-01" href="/">{cellValue}</a>
 		{:else if cellKey === 'result'}
-			<VotingResultTag class="m-0 whitespace-nowrap" result={cellValue} />
+			<VotingResultTag class="m-0 whitespace-nowrap" isLarge={true} result={cellValue} />
```

Comment:
Not sure if this is really necessary or not. As the old one seem to be noticeably larger.

Reply:
Let's keep it this way, you're doing right. By the way, for `isLarge={true}` we can just omit the value and write `isLarge`. This should be my last feedback :)